src/Integration/Binding/BindingController.cs

```diff
@@ -151,8 +150,20 @@ private void OnBindingFinished(ProjectInformation projectInformation, bool isFin
             if (isFinishedSuccessfully)
             {
                 this.host.VisualStateManager.SetBoundProject(projectInformation);
-                VsShellUtils.ActivateSolutionExplorer(this.ServiceProvider);
-                this.CheckForConflicts();
+
+                var conflictsController = this.host.GetService<IRuleSetConflictsController>();
+                conflictsController.AssertLocalServiceIsNotNull();
+
+                if (conflictsController.CheckForConflicts())
```

Comment:
Decided to introduce a separate controller instead of having all the code on the bindingcontroller. Having it separate means that we can call it from other places i.e. VS commad.